packages/common/src/chains/sepolia.json

```diff
@@ -0,0 +1,111 @@
+{
+  "name": "sepolia",
+  "chainId": 11155111,
+  "networkId": 11155111,
+  "defaultHardfork": "istanbul",
+  "consensus": {
+    "type": "pow",
+    "algorithm": "ethash",
+    "ethash": {}
+  },
+  "comment": "PoW test network",
+  "url": "https://github.com/ethereum/ropsten",
```

Comment:
Please update this URL with something fitting, a robust PR URL like https://github.com/ethereum/go-ethereum/pull/23730 would also be ok (this info is not read "by machine" anywhere, just some helper for someone digging into the chain files).